src/ansible_navigator/configuration_subsystem/navigator_post_processor.py

```diff
@@ -46,13 +49,64 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+class VolumeMountOption(Enum):
+    """Options that can be tagged on to the end of volume mounts.
+
+    Usually these are used for things like selinux relabeling, but there are
+    some other valid options as well, which can and should be added here as
+    needed. See ``man podman-run`` and ``man docker-run`` for valid choices and
+    keep in mind that we support both runtimes.
+    """
+
+    # Relabel as private
+    Z = "Z"
+
+    # Relabel as shared.
+    z = "z"  # pylint: disable=invalid-name
+
+
+@dataclass
+class VolumeMount:
+    """Describes EE volume mounts."""
+
+    # The name of the config option requiring this volume mount.
+    calling_option: str
+
+    # The source path of the volume mount.
+    src: str
+
+    # The destination path in the container for the volume mount.
+    dest: str
+
+    # Options for the bind mount.
+    options: Optional[List[VolumeMountOption]] = None
```

Comment:
I wonder if
```suggestion
    options: List[VolumeMountOption] = []
```
would help.
Or maybe you could use `__post_init__()` or have a custom property that would return `[]` if it's `None`.